src/nccl_ofi_rdma.c

```diff
@@ -5442,7 +5442,7 @@ static int release_ep(nccl_net_ofi_ep_t *base_ep)
 	return ret;
 }
 
-static int get_ep(nccl_net_ofi_device_t *base_dev, nccl_net_ofi_ep_t **base_ep)
+static inline int get_ep(nccl_net_ofi_device_t *base_dev, nccl_net_ofi_ep_t **base_ep)
```

Comment:
I think the checker is wrong here, but it's harmless, so whatever.

Reply:
We still have many other `static int` functions that do not hit this warning, and I think this has to do with ODR and the fact that both rdma and sendrecv have an `int get_ep(...);`

Reply:
yup, I think you're right.  That actually makes sense.  I would vote we prefix the names instead of inline (ie `nccl_net_ofi_rdma_device_get_ep`, but don't have super strong feelings.

Reply:
these should eventually merge back under a common generic method, prefer to keep it as is if you're okay with that.